packages/global/src/utils/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './assert.js';
 export * from './disposable.js';
 export * from './function.js';
+export * from './hover/index.js';
```

Comment:
I don't think it's appropriate to put this in the `global` package, which should only contain essential non-UI related primitives and utils.

Reply:
Now the API moved to `blocks/components/hover`

```
components
├── controller.ts <- HoverController
├── index.ts
├── middlewares
│   ├── basic.ts
│   └── safe-area.ts
├── types.ts
└── when-hover.ts
```